apps/perf-test-react-components/package.json

```diff
@@ -1,23 +1,17 @@
 {
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26827&commitId=3b279c128ca9c5e81c0034ebffc1ad5f605350f0&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1257 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Context Menu.chromium.png|16562| Changed|
|Card.Card Example Expandable.chromium.png|17227| Changed|
|Card.Card Example Selectable Grid.chromium.png|113007| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>